lib/api/multiObjectDelete.js

```diff
@@ -246,16 +374,16 @@ function multiObjectDelete(authInfo, request, log, callback) {
                     signatureAge: authParams.params.data.signatureAge,
                 },
                 parameterize: {
-                    specificResource: objects,
+                    specificResource: objects.map(entry => entry.key),
```

Comment:
Hm true, but this need to be handle in a subsequent pull request, we already have huge enought pull requests :)